addons/docs/src/frameworks/angular/__testfixtures__/doc-button/compodoc.snapshot

```diff
@@ -132,7 +132,7 @@ like <strong>bold</strong>, <em>italic</em>, and <code>inline code</code>.</p>
           "name": "click",
         },
       ],
-      "id": "component-InputComponent-8088cb533b6defa9d43d10841b59ce18",
+      "id": "component-InputComponent-14bbde487c28642f97f1f6c94b65ab31",
```

Comment:
I normally like to fold fixup commits to their associated change, but it isn't particularly obvious to me which caused this.

My guess is 3bfb2b673f (although I don't have the energy to investigate the details). And if someone confirms, I'm happy to fold it in...